torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1059,6 +1130,75 @@ def sample_inputs_pinverse(op_info, device, dtype, requires_grad=False):
            supports_tensor_out=False,
            sample_inputs_func=sample_inputs_pinverse,
            decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack]),
+    OpInfo('gather',
```

Comment:
method_tests for gather can be removed here:

https://github.com/pytorch/pytorch/blob/97c17b47720dd1afce88180b843c9182f1a665a9/torch/testing/_internal/common_methods_invocations.py#L2087